keyboards/easyconvert/readme.md

```diff
@@ -0,0 +1,22 @@
+# EASY-CONVERT
+
+## XXXL80 based on EASY-CONVERT
+![xxxl80 based on EASY-CONVERT](https://i.imgur.com/ODpqjla.jpg)
+
+* Keyboard Maintainer: [LiGuo Studio](https://github.com/mechlovin)
+* Hardware Supported: xxxl80, STM32L433 
```

Comment:
Should the keyboard not be named `xxxl80` instead of `easyconvert`?

Reply:
You havent answered the question here, so this should not be marked as resolved.

Reply:
Sorry, the keyboard should be named easyconvert

Reply:
> xxxl80 based on EASY-CONVERT

> * Hardware Supported: xxxl80

These really do not suggest that at all.

Reply:
I have modified the readme file.